infrastructure/infrastructure.tf

```diff
@@ -89,3 +94,40 @@ module "nmap_task" {
   ssm_source_stage              = "${local.ssm_source_stage}"
   transient_workspace           = "${!contains(var.known_deployment_stages, terraform.workspace)}"
 }
+
+module "nmap_task_scheduler" {
+  source = "github.com/ministryofjustice/securityanalytics-taskexecution/infrastructure/scheduler"
+
+  // It is sometimes useful for the developers of the project to use a local version of the task
+  // execution project. This enables them to develop the task execution project and the nmap scanner
+  // (or other future tasks), at the same time, without requiring the task execution changes to be
+  // pushed to master. Unfortunately you can not interpolate variables to generate source locations, so
+  // devs will have to comment in/out this line as and when they need
+  // source = "../../securityanalytics-taskexecution/infrastructure/scheduler"
+
+  app_name            = "${var.app_name}"
+  task_name           = "${var.task_name}"
+  scan_hosts          = "${var.scan_hosts}"
+  queue_url           = "${module.nmap_task.task_queue_url}"
+  queue_arn           = "${module.nmap_task.task_queue}"
+  transient_workspace = "${!contains(var.known_deployment_stages, terraform.workspace)}"
+}
+
+data "aws_iam_policy_document" "nmap_cwe_sqs_policy" {
```

Comment:
Since this has no other dependencies, it should be added to the task execution module, not the nmap

Reply:
Add this to the existing notify_topic_policy

Reply:
moved to trigger_queue in task execution